test/smoke.test.ts

```diff
@@ -175,11 +170,12 @@ describe("Smoke test", () => {
       title: "My test issue",
     });
 
-    expect(data).toStrictEqual({ ok: true });
```

Comment:
Same as https://github.com/octokit/rest.js/pull/305 and https://github.com/octokit/core.js/pull/563#discussion_r1258600840

Reply:
Addressed in https://github.com/octokit/core.js/issues/588